### PR TITLE
saving input

### PR DESCRIPTION
**Pull request by fabiooshiro**

### CODE REVIEW

File: internal/reader/adapter_v1.go [hunk 1]
```diff
@@ -266,16 +266,23 @@ func (a AdapterV1) GetInputs(
 		}
 		if where.Type != nil {
 			inputboxField := "InputBoxIndex"
-			defaultStr := "-1"
+			espressoStr := "-1"
```
Comment (fabiooshiro) [inline internal/reader/adapter_v1.go:269]: Why not the "type" field?

Comment (fabiooshiro) [reply internal/reader/adapter_v1.go:269]: cc @ZzzzHui

Comment (ZzzzHui) [reply internal/reader/adapter_v1.go:269]: because we can already differentiate the type based on their "InputBoxIndex" field, right? What do you think.

Comment (DiegoSena) [reply internal/reader/adapter_v1.go]: I'd rather go with the type field.. it is more clear. The input box field we'll have to map each value to its sequencer/DA in the code ..

Comment (ZzzzHui) [reply internal/reader/adapter_v1.go]: yes for multiple sequencers, it's more clear to have a type field

Comment (ZzzzHui) [reply internal/reader/adapter_v1.go]: > Why not the "type" field?

I thought the question was for me. But it's actually for Diego lol

**Review by sandhilt — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**